types/Cache_client/ScsClient.php

```diff
@@ -72,6 +72,48 @@ public function Delete(\Cache_client\_DeleteRequest $argument,
         $metadata, $options);
     }
 
+    /**
+     * @param \Cache_client\_KeysExistRequest $argument input argument
+     * @param array $metadata metadata
+     * @param array $options call options
+     * @return \Grpc\UnaryCall
+     */
+    public function KeysExist(\Cache_client\_KeysExistRequest $argument,
```

Comment:
everything in this dir is just generated code, yeah?

Reply:
Yes, everything under `types/` is generated.